olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```
@@ -393,7 +394,8 @@ spec:
               serviceAccountName: knative-openshift-ingress
               containers:
                 - name: knative-openshift-ingress
-                  image: $IMAGE_KNATIVE_OPENSHIFT_INGRESS
+                  # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress
```

```suggestion
                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.7.3:knative-openshift-ingress
```

(will need a change to openshift/release first I believe)

Same applies to other image refs.